<a href="https://colab.research.google.com/github/WJurgiel/Statistical-data-analysis/blob/PC2_statistical-hyphothesis-test/Rcode.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

Wagi [g] (7 prób): 142, 151, 148, 151, 145, 150, 141

Teoretyczna waga [g]: 150
α = 0.05

**Hipotezy: **

 * H0: μ=150* (przeciętna waga netto słoiczka kawy wynosi 150 g)

  *H1: μ≠150* (przeciętna waga netto słoiczka kawy różni się istotnie od wagi nominalnej)

Test statystyczny - parametryczny, pojedyncza próba, skala ilorazowa

In [1]:
kawa <- c(142, 151, 148, 151, 145, 150, 141)
t.test(kawa, mu=150)
# param_1 - vector with data
# param_2 - average value (H0 = 150)
# alpha is set by default as 0.05


	One Sample t-test

data:  kawa
t = -1.9704, df = 6, p-value = 0.0963
alternative hypothesis: true mean is not equal to 150
95 percent confidence interval:
 142.9542 150.7601
sample estimates:
mean of x 
 146.8571 


**wartość testowa** t = -1.9704

**p-value** = 0.0963 -> p > a, so there's no need to throw away H0. Average weight of jar with coffee doesn't differ significantly from 150g

# **TASK 2**
before:  27, 21, 34, 24, 30, 27, 33, 31, 22, 27

after:  29, 32, 29, 27, 31, 26, 35, 30, 29, 28.

alpha = 0.05

H0 no difference

H1 there's a difference

Parametrical test - average value, double sample, ratio scale


In [2]:
before <- c(27,21,34,24,30,27,33,31,22,27)
after <- c(29,32,29,27,31,26,35,30,29,28)
t.test(before, after, alternative = "greater", paired = TRUE)


	Paired t-test

data:  before and after
t = -1.4302, df = 9, p-value = 0.9068
alternative hypothesis: true mean difference is greater than 0
95 percent confidence interval:
 -4.563447       Inf
sample estimates:
mean difference 
             -2 


#test value = -1.4302
#p-value = 0.9068 > alpha
There's no fundamentals to reject H0, so it's true. Keep in mind that we could've committed 2nd kind mistake, what is not rejecting H0 even though it's false